server/world/recipes/dimensions.go

```diff
@@ -0,0 +1,17 @@
+package recipes
+
+// Dimensions make up the size of a shaped recipe.
+type Dimensions struct {
```

Comment:
Could just make this a `[2]int`, I wouldn't use int32s.

Reply:
Changed.